src/lib/viewers/controls/color-picker/__tests__/ColorPickerPalette-test.tsx

```diff
@@ -3,16 +3,16 @@ import { shallow, ShallowWrapper } from 'enzyme';
 import ColorPickerPalette from '../ColorPickerPalette';
 
 describe('ColorPickerPalette', () => {
-    const onColorSelect = jest.fn();
+    const colors = ['#fff'];
 
     const getWrapper = (props = {}): ShallowWrapper =>
-        shallow(<ColorPickerPalette onColorSelect={onColorSelect} {...props} />);
+        shallow(<ColorPickerPalette colors={colors} onSelect={jest.fn()} {...props} />);
 
     describe('render', () => {
-        test('should render the six color swatches', () => {
+        test('should render a single color swatch', () => {
```

Comment:
The coverage here seems a little lacking. Do we want to assert anything else about this component?

Reply:
I think we can assert that `onSelect` is called properly on button click.